packages/framer-motion/src/animation/GroupPlaybackControls.ts

```diff
@@ -44,6 +44,14 @@ export class GroupPlaybackControls implements AnimationPlaybackControls {
         this.setAll("speed", speed)
     }
 
+    get duration() {
```

Comment:
```suggestion
    /* The duration is the duration of time calculated for the active part of the animation without delay or repeat, which may be added as an extra prop at a later date. */
    get duration() {
```

Reply:
I'll put a comment in the underlying types so it applies to all implementations